### ISSUE TITLE
Initial contents of Redis are not available

### ISSUE THREAD

**Issue by flavio** [vertexclique/docker-volume-redis#4]
Hi, while testing the code I found that the initial contents of Redis are not available inside of the container
## How to reproduce the problem
- Start with an empty Redis database
- Add a couple of key/value tuples.
- Start the docker-volume-redis daemon
- Start an interactive bash session inside of a docker container using the Redis volume plugin
- Within the container type: `ls /redis`

Expected outcome:
- The keys added during the 2nd step should be available as simple text files.

Actual result:
- The directory is empty.

What do you think is causing the issue? You don't have to provide a bug fix yet, we can just talk about it.

**Comment by vertexclique** [comment]
I am not watching changes in redis store, so i am not acquiring KV pairs when they have added in redis. If I watch all keys continuously and write them, this should be solved, or subscribe to key event notifications from redis.

**Comment by flavio** [comment]
I don't think this is going to solve the problem. Note that Redis has already data inside of it before the `docker-volume-redis` daemon process is started. This issue is about the already existing contents of Redis missing from the volume.

**Comment by vertexclique** [comment]
Ok i can create all the files with reading all redis key-values when volumes are mounted.

**Comment by flavio** [comment]
Yes, that's the answer I was looking for. In fact the 1st problem of issue #5 is just a duplicate of this one.
